modules/ROOT/pages/usage-metrics.adoc

```diff
@@ -106,28 +114,293 @@ Execution types associated with the processed action
 Processed Pages::
 Sum of pages processed
 
-
 == Mule Runtime Usage
 
-Mule Runtime usage reports show this information in tables and cards:
+MuleSoft captures usage data for Mule flows, Mule messages, and data throughput, but not all of this data is aggregated in usage reports.  
+
+To track Mule message usage, the runtime report counts the number of times a Mule event source triggers a Mule message. You can view the number of these messages in a given day or month by business group, environment, and application. 
+
+To calculate usage, MuleSoft meters and aggregates the number of messages daily and monthly. After a message is triggered, the report doesn't track changes to the message because the message is processed within the application’s flows.
+
+Data throughput is the network I/O bytes produced by the infrastructure that runs the Mule runtime engine running the Mule application. 
+
+To calculate usage, MuleSoft tracks GB usage and aggregates the total daily and monthly GBs.
+
+For select customers in the US control plane, MuleSoft offers a pricing and packaging model for Anypoint Platform that allots a number of Mule flows, Mule messages, and data throughput (measured as network I/O bytes). Not all types of Mule flows and messages count toward the allotments in a package.
+
+=== Mule Runtime Usage Tables
+
+The usage information shown in the usage report tables changes depending on the view you are in.
```

Comment:
```suggestion
The usage information shown in the usage report tables changes depending on your view.
```

Reply:
Can you be "in" a view?

Reply:
I think they can. They can switch between views. "depending on your view" sounds like "depending on what you think..."